packages/core/src/RPA/core/windows/elements.py

```diff
@@ -0,0 +1,122 @@
+from pathlib import Path
+from typing import Dict, List, Optional
+
+from RPA.core.vendor.deco import keyword as method
+from RPA.core.windows.context import WindowsContext
+from RPA.core.windows.helpers import IS_WINDOWS
+from RPA.core.windows.locators import Locator, MatchObject, WindowsElement
+
+if IS_WINDOWS:
+    import uiautomation as auto
+    from uiautomation.uiautomation import Control
+
+
+StructureType = Dict[int, List[WindowsElement]]
+
+
+class ElementMethods(WindowsContext):
+    """Keywords for listing Windows GUI elements."""
+
+    def _save_child_image(
```

Comment:
The `child` is prob not relevant information for the name of this method - just `_save_image`

And needs to be synced with RPA.Windows.Screenshot keyword.